projects/1.16.1/assets/cazfps-the-dead-sea/cazfps_the_dead_sea/lang/zh_cn.json

```
@@ -0,0 +1,93 @@
+{
+  "advancements.tds.descr": "惊恐维度！",
+  "block.cazfps_the_dead_sea.steeltrapdoor": "钢活板门",
+  "block.cazfps_the_dead_sea.concretwall": "混凝土墙",
+  "item.cazfps_the_dead_sea.forestghost": "死森幽魂 刷怪蛋",
+  "item.cazfps_the_dead_sea.deadskeleton": "死亡骷髅 刷怪蛋",
+  "block.cazfps_the_dead_sea.steelwall": "钢墙",
+  "item.cazfps_the_dead_sea.deadclown": "死亡小丑 刷怪蛋",
+  "item.cazfps_the_dead_sea.oldman_spawn_egg": "可怖老人 刷怪蛋",
+  "block.cazfps_the_dead_sea.deadstonestairs": "死之石楼梯",
+  "advancements.tds.title": "死亡之海",
+  "entity.cazfps_the_dead_sea.deadwalker": "死亡游荡者",
+  "block.cazfps_the_dead_sea.deaddirt": "死亡土",
+  "item.cazfps_the_dead_sea.bigfoot_spawn_egg": "大脚巨人 刷怪蛋",
+  "block.cazfps_the_dead_sea.pathblock": "石路",
+  "key.mcreator.tdsk": "死亡之海 菜单",
+  "block.cazfps_the_dead_sea.deadgrass": "死亡草",
+  "item.cazfps_the_dead_sea.crosshearth": "Cross hearth amo",
+  "block.cazfps_the_dead_sea.deadtreeplank": "死亡之树木板",
+  "entity.cazfps_the_dead_sea.deadwitch": "死亡女巫",
+  "block.cazfps_the_dead_sea.steelslab": "钢台阶",
+  "block.cazfps_the_dead_sea.dsslab": "死亡石台阶",
+  "block.cazfps_the_dead_sea.deadsand": "死亡沙",
+  "block.cazfps_the_dead_sea.dps": "死亡之树木板楼梯",
+  "block.cazfps_the_dead_sea.deadtreebrench": "死亡之树树枝",
+  "block.cazfps_the_dead_sea.annimateddeadtreelog": "死亡之树原木（活）",
+  "item.cazfps_the_dead_sea.deadclown_spawn_egg": "死亡小丑 刷怪蛋",
+  "block.cazfps_the_dead_sea.dtps": "死亡之树木台阶",
+  "block.cazfps_the_dead_sea.deadtreeslab": "死亡之树台阶",
+  "block.cazfps_the_dead_sea.handplant": "死之草",
```

统一一下风格可能更好（死亡之海、死亡之树、死亡之… 一以贯之）
```suggestion
  "block.cazfps_the_dead_sea.handplant": "死亡之草",
```